client-datastore-cassandra/src/main/resources/schema-cassandra.cql

```diff
@@ -26,7 +26,8 @@ CREATE TABLE IF NOT EXISTS tip_wlan_keyspace.client_by_mac_string (
     PRIMARY KEY (customerId, macAddressString)
 ) WITH comment='Index Table to look up wireless client details by macAddress in text form used by TIP WLAN CloudSDK';
 
-CREATE CUSTOM INDEX IF NOT EXISTS idx_client_macAddressString ON tip_wlan_keyspace.client_by_mac_string (macAddressString) USING 'org.apache.cassandra.index.sasi.SASIIndex' WITH OPTIONS = {'mode': 'CONTAINS', 'analyzer_class': 'org.apache.cassandra.index.sasi.analyzer.StandardAnalyzer', 'case_sensitive': 'false'};
+DROP INDEX IF EXISTS idx_client_macAddressString;
```

Comment:
did you try executing this command?
for me it does not work without "tip_wlan_keyspace." prefix.
create index works, but delete index does not.

Reply:
The command does work for me, but I will specify the keyspace as well. 
For me, both worked

Reply:
it works if you explicitly call 
USE tip_wlan_keyspace;

I'm not sure if the caller of this scql script would do that, so it's better to put the prefix in place.